File: nodejs/01-estrutura-sequencial/ex02.js
/*
 * Exercício 02
 *
 * Faça um Programa que peça um número e então mostre a mensagem
 * "O número informado foi [número].
 */

import * as readline from 'node:readline'
import {stdin as input, stdout as output} from 'node:process'

const rlp = readline.createInterface({input, output})

rlp.question("Digite um número: ", (numero) => {
    console.log(`O número informado foi [${parseInt(numero)}]`)
    rlp.close()
})

File: dotnet/01.EstruturaSequencial/Ex02.cs
namespace EstruturaSequencial;

static class Ex02
{
    /// <summary>
    /// Faça um Programa que peça um número e então mostre a mensagem
    /// "O número informado foi [número]."
    /// </summary>
    public static void Run()
    {
        Console.Write("Digite um número: ");
        int numero = int.Parse(Console.ReadLine()!);
        Console.Write($"O número informado foi [{numero}]");
    }
}